src/skin.cpp

```diff
@@ -387,7 +387,10 @@ void SkinInfillAreaComputation::generateSkinInsets(SkinPart& skin_part)
         }
 
         // optimize polygons: remove unnecessary verts
-        skin_part.insets[inset_idx].simplify();
+        const ExtruderTrain& train_wall = mesh.settings.get<ExtruderTrain&>(inset_idx == 0 ? "wall_0_extruder_nr" : "wall_x_extruder_nr");
+        const coord_t maximum_resolution = train_wall.settings.get<coord_t>("meshfix_maximum_resolution");
+        const coord_t maximum_deviation = train_wall.settings.get<coord_t>("meshfix_maximum_deviation");
+        skin_part.insets[inset_idx].simplify(maximum_resolution, maximum_deviation);
```

Comment:
I can't believe we didn't do this before...

Reply:
I know, right?

Reply:
It's not _quite_ as bad as it looks, as it gets simplified with the right parameters elsewhere .... but I'm pretty sure there are alterations to those polygons in the meantime.

Reply:
There's also a few instances I didn't get to, because they do raft, support, infill etc, but we might want to have a look later at those as well.